config/eslintrc_react.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
         'react/default-props-match-prop-types': 1,
         'react/display-name': 0, // Auto covered by jsx transformer.
         'react/forbid-component-props': 1,
+        'react/forbid-dom-props': 1,
```

Comment:
@kanufy  Thank you for updating you patch!

Ugh, [this rule requires the black list](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/forbid-dom-props.md) for attributes. So you need add it to prevent to use `id` attribute for plain DOM node.

Could you address it?